.github/policies/resourceManagement.yml

```
@@ -0,0 +1,518 @@
+            pattern: Copied from original issue
+            isRegex: False
+        - bodyContains:
+            pattern: Issue moved from
+            isRegex: False
+      then:
+      - addLabel:
+          label: Transferred issue
+      - addLabel:
+          label: Triage:Untriaged
+      description: "[Manage \"WaitingFor\" labels] [4-1] Tag \"Transferred issue\" if it's transferred from vsfeedback or other github repo."
+    - if:
+      - payloadType: Issue_Comment
+      - isAction:
+          action: Created
+      - hasLabel:
+          label: WaitingForCustomer
+      - isOpen
+      - or:
+        - and:
+          - hasLabel:
+              label: Transferred issue
+          - not:
+              activitySenderHasPermission:
+                permission: Write
+        - isActivitySender:
+            issueAuthor: True
+      then:
+      - removeLabel:
+          label: WaitingForCustomer
+      - addLabel:
+          label: WaitingForClientTeam
+      - removeLabel:
+          label: Status:No recent activity
+      description: "[Manage \"WaitingFor\" labels] [4-2] Replace tag \"WaitingForCustomer\" with \"WaitingForClientTeam\" when the author comments on an issue. Also remove `Status:No recent activity` if it's been set."
+    - if:
+      - payloadType: Issues
+      - isAction:
+          action: Closed
+      - or:
+        - hasLabel:
+            label: WaitingForClientTeam
+        - hasLabel:
+            label: WaitingForCustomer
+      then:
+      - removeLabel:
+          label: WaitingForClientTeam
+      - removeLabel:
+          label: WaitingForCustomer
+      description: '[Manage "WaitingFor" labels] [4-3] Remove any "WaitingFor" label when the issue is closed'
+    - if:
+      - payloadType: Issue_Comment
+      - isAction:
+          action: Created
+      - hasLabel:
+          label: WaitingForClientTeam
+      - isOpen
+      - or:
+        - activitySenderHasPermission:
+            permission: Write
+        - activitySenderHasPermission:
+            permission: Admin
+      then:
+      - removeLabel:
+          label: WaitingForClientTeam
+      - addLabel:
+          label: WaitingForCustomer
+      description: '[Manage "WaitingFor" labels] [4-4] Replace tag "WaitingForClientTeam" with "WaitingForCustomer" when client team comments on an issue.'
+    - if:
+      - payloadType: Pull_Request
+      - not:
+          isAction:
+            action: Closed
+      - hasLabel:
+          label: Status:No recent activity
+      then:
+      - removeLabel:
+          label: Status:No recent activity
+      description: '[stale PR] [5-2] Remove "Status:No recent activity" if there is any activity.'
+    - if:
+      - payloadType: Issue_Comment
+      - hasLabel:
+          label: Status:No recent activity
+      then:
+      - removeLabel:
+          label: Status:No recent activity
+      description: '[stale PR] [5-3] Remove "Status:No recent activity" if there is any comment.'

```

The previous configuration was limited to pull requests via the `"eventType": "pull_request"` configuration. This will bring back that condition. /cc @JohannesLampel

```suggestion
    - if:
      - payloadType: Issue_Comment
      - isPullRequest
      - hasLabel:
          label: Status:No recent activity
      then:
      - removeLabel:
          label: Status:No recent activity
      description: '[stale PR] [5-3] Remove "Status:No recent activity" if there is any comment.'
```